designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview-field-recon.component.html

```diff
@@ -189,8 +189,15 @@ <h3 class="text-center">
                                 <div class="entity-meta-data" ng-if="!$ctrl.readOnly && mission.value.dois.toString() === ''">
                                     <span>(Appears when published)</span>
                                 </div>
-                                <div class="entity-meta-data" ng-if="$ctrl.readOnly || mission.value.dois.toString() !== ''">
-                                    <span>{{ $ctrl.dateCreated | date:'MM-dd-yyyy' }}</span>
+                                <div class="entity-meta-data" ng-if="!$ctrl.readOnly && mission.value.dois.toString() !== ''"> 
+                                    <div ng-repeat = "list in $ctrl.dateCreated">
```

Comment:
Will there ever be a situation where there are multiple dates associated with one DOI here? If not then `dateList` can probably be an object that looks like `{[doi]: dateCreated}` and you can do an O(1) lookup instead of a loop.